### ISSUE TITLE
Loop in elements

### ISSUE THREAD

**Issue by shahenazmonia** [lotus-1/movie-Giphy-#15]
You should make loop to create className or ids for the images and put the image inside ```src``` tag in dom instead of creating many elements in ```html``` and get specific data from the api
```js   
   document.getElementById("giphy1").src = data.data[0].images.original.url;
      document.getElementById("giphy2").src = data.data[1].images.original.url;
      document.getElementById("giphy3").src = data.data[2].images.original.url;
```

Good job guys ❤️ good luck in the coming projects ❤️

**Comment by enasskmal** [comment]
yes we did this. and we create anther file called index.js . now we should change the name of the anther file that called main.js to DOM. thank you
